src/tests/simulation/Cargo.toml

```diff
@@ -43,6 +43,7 @@ serde_derive = "1.0.152"
 serde_json = "1.0.103"
 sqllogictest = "0.15.1"
 tempfile = "3"
+tikv-jemallocator = { git = "https://github.com/yuhao-su/jemallocator.git", branch = "yuhao/prof", features = ["profiling"] }
```

Comment:
This should only be a provisional solution. Can you please merge it into upstream (tikv/jemallocator)?

Reply:
Sure, should I merge this pr after I merger jenallocator?

Reply:
Not necessary I think, but make sure to fix it later ☺️